gke-gs-bucket-backup/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM google/cloud-sdk@sha256:3eb726653fe2f83952982bd47607c563e387bfc791f7d431f7f
 
 COPY gs_bucket_sync.py /usr/local/bin
 
-RUN adduser --disabled-password --uid 1000 gcloud_user
+RUN adduser --disabled-password --uid 1000 --gecos "" gcloud_user
```

Comment:
Off-topic, no changes requested: I'm trying to train myself to use `ARG UID=1000`, then `--uid ${UID}` in container specs so we can change it later if need be. Curious if you'd prefer a different pragma, or if you don't think it's worth the trouble.

Reply:
That would be good! Was just trying to do the smallest change here to fix the build. Will follow up.

Reply:
Added that as well.